### PR TITLE
Fix/fullscreen issue

### PR DESCRIPTION
**Pull request by antkiewicz**

### CODE REVIEW

File: apps/codelab/src/app/components/buttons-nav-bar/menu-fullscreen-widget/menu-fullscreen-widget.component.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <button class="menu-bar-btn" (click)="openFullScreen()">
-  <img alt="GitHub" src="/assets/images/icons/fullscreen.png" />
+  <img alt="Toggle Fullscreen" src="/assets/images/icons/fullscreen.png" />
```
Comment (NothingEverHappens) [inline apps/codelab/src/app/components/buttons-nav-bar/menu-fullscreen-widget/menu-fullscreen-widget.component.html]: Haha, good catch

File: libs/utils/src/lib/full-screen-mode/full-screen-mode.service.ts [hunk 2]
```diff
@@ -1,18 +1,75 @@
...
+import { DOCUMENT } from '@angular/common';
+
+interface FullscreenAPIMap {
+  requestFullscreen: string;
+  exitFullscreen: string;
+  fullscreenElement: string;
+}
+
+// Fullscreen API fallback map
+const FN_MAP: { [P in keyof FullscreenAPIMap]: string[] } = {
```
Comment (NothingEverHappens) [inline libs/utils/src/lib/full-screen-mode/full-screen-mode.service.ts]: Is there like a small library for this we could use? Or maybe create a small lib if that does not exist? Seems useful and your code seems pretty generic

File: libs/utils/src/lib/full-screen-mode/full-screen-mode.service.ts [hunk 3]
```diff
@@ -1,18 +1,75 @@
...
 })
 export class FullScreenModeService {
 
-  constructor() { }
+  private fn: FullscreenAPIMap;
+
+  constructor(
+    @Inject(DOCUMENT) private document: Document
+  ) {
+    this.fn = Object.keys(FN_MAP).reduce((fns, key) => {
```
Comment (NothingEverHappens) [inline libs/utils/src/lib/full-screen-mode/full-screen-mode.service.ts]: Can we find more specific name and move this out of constructor?

File: libs/utils/src/lib/full-screen-mode/full-screen-mode.service.ts (hunk 2 above)
Comment (antkiewicz) [reply libs/utils/src/lib/full-screen-mode/full-screen-mode.service.ts]: @NothingEverHappens We could replace this whole service with this: https://www.npmjs.com/package/angular-bigscreen

**Pull request merged**